File: services/lucy-dotnet/src/Lucy.AuthPayment.Api/Modules/Payment/Models/LiveAccessTicket.cs
namespace lucy_auth_payment.Modules.Payment.Models;

public class LiveAccessTicket
{
    public string TicketId { get; set; } = null!;
    public string RoomId { get; set; } = null!;
    public string UserId { get; set; } = null!;
    public string? PurchaseId { get; set; }
    public string TicketStatus { get; set; } = "ACTIVE";
    public DateTime CreatedAt { get; set; } = DateTime.Now;
}

File: lucy-web-stack/lucy-backend/src/main/java/com/lucy/backend/payment/entity/LiveAccessTicket.java
package com.lucy.backend.payment.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "LiveAccessTickets")
@Data
public class LiveAccessTicket {
    @Id
    @Column(name = "TicketId", length = 50)
    private String ticketId;

    @Column(name = "RoomId", length = 50)
    private String roomId;

    @Column(name = "UserId", length = 50, nullable = false)
    private String userId;

    @Column(name = "WalletId", length = 50)
    private String walletId;

    @Column(name = "TicketStatus", length = 30, nullable = false)
    private String ticketStatus = "ACTIVE";
}
